api/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/containerd/containerd/api
 go 1.21
 
 require (
-	github.com/containerd/ttrpc v1.2.3
+	github.com/containerd/ttrpc v1.2.4
 	github.com/containerd/typeurl/v2 v2.1.1
 	github.com/opencontainers/image-spec v1.1.0
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240401170217-c3f982113cda
```

Comment:
FWIW; this one is ahead of the 1.7 api module. Not sure if problematic for 1.7 consumers, but we _could_ (if it is) downgrade this to a lower "minimum required" version; https://github.com/googleapis/go-genproto/compare/b8732ec3820d...c3f982113cda